File: src/main/java/com/noname/duyuru/app/json/models/ChatMember.java
package com.noname.duyuru.app.json.models;

import com.fasterxml.jackson.databind.PropertyNamingStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import com.noname.duyuru.app.jpa.models.User;
import lombok.Data;

@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
@Data
public class ChatMember {
    User user;
    String status;
}

File: chattools/src/main/kotlin/aytackydln/chattools/telegram/dto/models/ChatMember.kt
package aytackydln.chattools.telegram.dto.models

import aytackydln.chattools.telegram.TelegramChat
import com.fasterxml.jackson.databind.PropertyNamingStrategies
import com.fasterxml.jackson.databind.annotation.JsonNaming
import lombok.Data
import lombok.RequiredArgsConstructor

@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
class ChatMember(
    var user: TelegramChat,
    var status: String,
)
